stories/index.stories.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { withKnobs, number, text } from '@storybook/addon-knobs/react'
+import { withInfo } from '@storybook/addon-info';
```

Comment:
This can go in the `main.js` file like knobs - and the addon gets added everywhere so you don't need the `.addDecorator()` below

Reply:
@maeriens I'll try moving it, but guessing it won't fix whatever is preventing it from working currently - any ideas on that?

Reply:
I am guessing what you want us something like this below - it is a bit tricky with the current setup because you'd have to inject the story source and would be easier by splitting up the stories I guess. The simple code I uploaded in [here](https://github.com/maeriens/react-sticky-table/tree/storybook-addon-info)
![table](https://user-images.githubusercontent.com/5074186/82783089-2b4a5080-9e34-11ea-93a5-5f0a070b0a98.gif)